gui/system/migrations/0052_move_syslog_from_settings_model_to_advanced_model.py

```diff
@@ -0,0 +1,40 @@
+from django.db import migrations
```

Comment:
Please squash all the migrations.

Also you're adding the field to Advanced but not removing from Settings.